### ISSUE TITLE
One-directional messaging bus service bletchley fabric

### ISSUE THREAD

**Issue by ygizhitsa** [Azure/azure-blockchain-projects#53]
Cryptlets  is very powerful design pattern that covers a lot of off-chain interactions scenarios . However, please consider adding a somekind one-direction messaging service to be a part of the Blockchain Fabric. This will allow to simplify one-way  pub/sub type messaging  implementation and will easy attestation and  trust concerns ( reason: one directional -  blockachain-to-off-chain) .  In the use case where completion of the contract is logical event that is published into the accessible to the blockchain EventHub and consumed by off-chain subscribers   for the off-chain reconciliation process, such messaging service would reduce implementation efforts and would effect ROI

**Comment by marleyg** [comment]
Yes, that one of the patterns, along with cross chain/platform asset transfer and logic sharing, proofs sent to public chains, etc.  More to come soon.
